srcpkgs/epson-inkjet-printer-escpr2/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'epson-inkjet-printer-escpr2'
+pkgname=epson-inkjet-printer-escpr2
+version=1.1.15
+revision=1
+archs="x86_64 i686 aarch64 armv7l"
+create_wrksrc=no
+build_wrksrc=${pkgname}-${version}
+build_style=configure
+configure_script=./configure
```

Comment:
Unnecessary

Reply:
Pretty sure the build_style and configure script is not unnecessary ... You sure ?

Reply:
`configure_script=./configure` is the default, so it is unnecessary and noisy to define in template.

Inline comments relate to last line, other lines are added by github for context.

Reply:
Ok , the docs don't mention that ( I think) , nevertheless the next commit is going to have that line removed